corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -598,7 +598,7 @@ def get_domain_stats_data(domains, datespan, interval,
 
 def commtrack_form_submissions(domains, datespan, interval,
         datefield='received_on'):
-    mobile_workers = UserES().exclude_source().mobile_users().show_inactive().run().doc_ids
+    mobile_workers = UserES().mobile_users().show_inactive().get_ids()
```

Comment:
Seems like this should filter on domains, and use `get_mobile_users`... Not blocking, this file could use a once over for filtering by domains. I think we're returning lots of docs here that aren't needed